runtime/flang/stime3f.c

```diff
@@ -18,9 +18,9 @@
 int ENT3F(STIME, stime)(int *tp)
 {
   int i;
-  time_t t = *tp;
-
-  if ((i = stime(&t)))
+  struct timespec ts = {};
```

Comment:
LGTM.

I observed the following warning with gcc -pedantic.
warning: ISO C forbids empty initializer braces [-Wpedantic]
   struct timespec ts = {}

Reply:
Thank you for the reviews. @kiranchandramohan  I have fixed the empty initializer warning now.